src/Nethermind/Nethermind.Specs.Test/ChainSpecStyle/ChainSpecBasedSpecProviderTests.cs

```diff
@@ -821,6 +825,199 @@ public static IEnumerable BlockNumbersAndTimestampsNearForkActivations
         }
     }
 
+    [TestCaseSource(nameof(BlobScheduleActivationsTestCaseSource))]
+    public void Test_BlobScheduleActivations((string fork, ulong timestamp, ulong? blobSetting)[] settings, ulong[] expectedActivationSettings)
```

Comment:
```suggestion
    public void Test_BlobScheduleActivations((string fork, ulong timestamp, ulong? blobSettings)[] settings, ulong[] expectedActivationSettings)
```
The name `blobSettings` is a bit confusing, why is this a ulong?